Packs/CortexXDR/ReleaseNotes/4_10_19.md

```diff
@@ -0,0 +1,10 @@
+
+#### Playbooks
+
+##### Cortex XDR Alerts Handling
+
+- hardcoded value for "UserEngagementThreshold" input has been removed. The input value can now be adjusted through the sub-playbook default values.
+##### Cortex XDR - Possible External RDP Brute-Force
+
+- Description fix for "UserEngagementThreshold" input
```

Comment:
Decide wether to end the sentence with a full stop or not. One has and the other doesn't.